### ISSUE TITLE
Chinese font cant display whole content

### ISSUE THREAD

**Issue by colin4k** [ppisljar/pdf_translator#14]
If change to Chinese fonts, the text content cant be displayed totally, some content would be cut.
[translated.pdf](https://github.com/user-attachments/files/16200649/translated.pdf)

**Comment by gesen2egee** [comment]
![image](https://github.com/user-attachments/assets/0614da7a-a0e2-4d81-aaa8-0efb2cc3efa8)
It seems to be related to the textwrap_local section, which was originally set for Japanese, so there is a discrepancy in the width for Chinese. I modified it like this; the output isn't very aligned but won't truncate the text. There might be a better way to adjust it, but I'm not sure.

**Comment by colin4k** [comment]
> ![image](https://private-user-images.githubusercontent.com/79357052/348952323-0614da7a-a0e2-4d81-aaa8-0efb2cc3efa8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzNTgzMzgsIm5iZiI6MTcyMTM1ODAzOCwicGF0aCI6Ii83OTM1NzA1Mi8zNDg5NTIzMjMtMDYxNGRhN2EtYTBlMi00ZDgxLWFhYTgtMGVmYjJjYzNlZmE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE5VDAzMDAzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgzMTAwMzFjMjE2NzM5MjI0NTYzNDY2YzFhOWYyZTdjOTM2MDAxZmQ3YTZhNTRjMDQzZTI5OGNkZmIxMmY3ZDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pKQZC0v2mx_jt8CgweSm75JC_xF85CM4pl54diFxkkU) It seems to be related to the textwrap_local section, which was originally set for Japanese, so there is a discrepancy in the width for Chinese. I modified it like this; the output isn't very aligned but won't truncate the text. There might be a better way to adjust it, but I'm not sure.
> 
> #EDIT All set to 2.4 is good

No, your code didn't solve the problem, the text still was truncated. You can see the attachment.
[translated-3.pdf](https://github.com/user-attachments/files/16301633/translated-3.pdf)

**Comment by colin4k** [comment]
There are 

> ![image](https://private-user-images.githubusercontent.com/79357052/348952323-0614da7a-a0e2-4d81-aaa8-0efb2cc3efa8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzNTgzMzgsIm5iZiI6MTcyMTM1ODAzOCwicGF0aCI6Ii83OTM1NzA1Mi8zNDg5NTIzMjMtMDYxNGRhN2EtYTBlMi00ZDgxLWFhYTgtMGVmYjJjYzNlZmE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE5VDAzMDAzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgzMTAwMzFjMjE2NzM5MjI0NTYzNDY2YzFhOWYyZTdjOTM2MDAxZmQ3YTZhNTRjMDQzZTI5OGNkZmIxMmY3ZDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pKQZC0v2mx_jt8CgweSm75JC_xF85CM4pl54diFxkkU) It seems to be related to the textwrap_local section, which was originally set for Japanese, so there is a discrepancy in the width for Chinese. I modified it like this; the output isn't very aligned but won't truncate the text. There might be a better way to adjust it, but I'm not sure.
> 
> #EDIT All set to 2.4 is good
There were many Chinese words totally being cut and hidden, so maybe you think it wasn't just very aligned.

**Comment by colin4k** [comment]
> ![image](https://private-user-images.githubusercontent.com/79357052/348952323-0614da7a-a0e2-4d81-aaa8-0efb2cc3efa8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzNTgzMzgsIm5iZiI6MTcyMTM1ODAzOCwicGF0aCI6Ii83OTM1NzA1Mi8zNDg5NTIzMjMtMDYxNGRhN2EtYTBlMi00ZDgxLWFhYTgtMGVmYjJjYzNlZmE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE5VDAzMDAzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgzMTAwMzFjMjE2NzM5MjI0NTYzNDY2YzFhOWYyZTdjOTM2MDAxZmQ3YTZhNTRjMDQzZTI5OGNkZmIxMmY3ZDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pKQZC0v2mx_jt8CgweSm75JC_xF85CM4pl54diFxkkU) It seems to be related to the textwrap_local section, which was originally set for Japanese, so there is a discrepancy in the width for Chinese. I modified it like this; the output isn't very aligned but won't truncate the text. There might be a better way to adjust it, but I'm not sure.
> 
> #EDIT All set to 2.4 is good

OK, I knew. All set to '2.4' would still be truncated, the config in your photo is good.

**Comment by gesen2egee** [comment]
good
maybe because we used different font/pdf layout
